src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
+
+{% if lastPage > 1 %}
+
+    {# the number of first and last pages to be displayed #}
+    {% set extremePagesLimit = 3 %}
+
+    {# the number of pages that are displayed around the active page #}
+    {% set nearbyPagesLimit = 2 %}
+
+    <div class="pagination pagination-centered">
+        <ul>
+          {% if currentPage > 1 %}
+            <li><a href="{{ paginationPath }}{{ currentPage-1 }}">&larr;</a></li>
+
+            {% for i in range(1, extremePagesLimit) if ( i < currentPage - nearbyPagesLimit ) %}
+                <li><a href="{{ paginationPath }}{{ i }}">{{ i }}</a></li>
+            {% endfor %}
+
+            {% if extremePagesLimit + 1 < currentPage - nearbyPagesLimit %}
+                <li class="disabled"><a href="#">...</a></li>
+            {% endif %}
+
+            {% for i in range(currentPage-nearbyPagesLimit, currentPage-1) if ( i > 0 ) %}
+                <li><a href="{{ paginationPath }}{{ i }}">{{ i }}</a></li>
+            {% endfor %}
+          {% elseif showAlwaysFirstAndLast %}
+            <li><a href="#">Previous</a></li>
+          {% endif %}
+
+        <li class="active"><a href="#">{{ currentPage }}</a></li>
+
+        {% if currentPage < lastPage %}
+            {% for i in range(currentPage+1, currentPage + nearbyPagesLimit) if ( i <= lastPage ) %}
```

Comment:
Having the `if` and the for on the same line is a bit confusing. Does this mean the for loop only runs if the condition is met or does the for loop then only output things if the condition is met? I'd prefer them on separate lines